selene/src/json_output.rs

```diff
@@ -1,12 +1,23 @@
+use std::io;
 use codespan_reporting::diagnostic::{
     Diagnostic as CodespanDiagnostic, Label as CodespanLabel, LabelStyle, Severity,
 };
 use serde::Serialize;
+use termcolor::StandardStream;
+use std::io::Write;
 
 #[derive(Serialize)]
 #[serde(tag = "type")]
 pub enum JsonOutput {
     Diagnostic(JsonDiagnostic),
+    Summary(JsonSummary),
+}
+
+#[derive(Serialize)]
+pub struct JsonSummary {
+    pub(crate) errors: usize,

```

Comment:
No need for `pub(crate)`, just `pub` is acceptable

Reply:
This was a leftover of when I was creating `JsonSummary` in `main`, so I'm going to remove the `pub` completely, in line with the other Json structs.